packages/oui-datagrid/README.md

```diff
@@ -91,6 +91,38 @@
 Clicked row action 1: <span ng-if="$ctrl.action1Row">{{$ctrl.action1Row.lastName}}, {{$ctrl.action1Row.firstName}}</span>
 ```
 
+### Global Actions
+
+```html:preview
+<oui-datagrid rows="$ctrl.data" page-size="5" global-actions>
```

Comment:
Wouldn't it be best to name the parameter "selectable-rows" ?

Reply:
sounds good

Reply:
Don't forget to add the details in the API table below for this attribute

Reply:
Were it only me, I would have named it `allows-row-selection` but I know you guys won't like it 😄

Reply:
In this way, we keep our logic of *able attributes